frontend/javascripts/viewer/store.ts

```diff
@@ -507,6 +507,7 @@ type UiInformation = {
   readonly borderOpenStatus: BorderOpenStatus;
   readonly theme: Theme;
   readonly isWkReady: boolean;
+  readonly isUiReady: boolean;
```

Comment:
the difference between wk-ready and ui-ready is not obvious, imo. what do you think about renaming `isWkReady` to `isWkInitialized`? if we want to keep the naming, there should be a comment which explains the semantics.